google/cloud/bigtable/instance_admin_client.h

```diff
@@ -87,7 +88,11 @@ class InstanceAdminClient {
   // them protected, so the mock classes can override them, and then make the
   // classes that do use them friends.
  protected:
-  friend class InstanceAdmin;
+  void MakeConnection(Options options) {
```

Comment:
This function seems to be used only once?  Should we just remove it and do the work in the header?  Alternatively, a static function that returns the value to put in `connection_`?

Reply:
I moved `connection_` into the implementation class. The base class has a virtual accessor (for `InstanceAdmin` to use).

My plan is to make this a pure virtual function. I do not feel bad about adding a pure virtual function, because we are going to break the class anyway. I did not make it a pure virtual function at this moment because I did not feel like making changes in derived classes (`mock_instance_admin_client.h`, `logging_instance_admin_client.h`).

I dismissed your approval. PTAL.